celer/tests/test_mtl.py

```diff
@@ -3,7 +3,8 @@
 
 import numpy as np
 from numpy.linalg import norm
-from numpy.testing import assert_allclose, assert_array_less, assert_array_equal
+from numpy.testing import (assert_allclose, assert_array_less,
```

Comment:
avoid these kind of changes that make the diff longer than it should

In this repo the line length limit is 88 (à la black)

you can edit your vscode setting to avoir autopep8 doing this on save